chat-plugins/mafia.js

```diff
@@ -1724,6 +2074,11 @@ const commands = {
 			const game = /** @type {MafiaTracker} */ (room.game);
 			if (game.closedSetup) return this.errorReply(`You cannot show roles in a closed setup.`);
 			if (!this.runBroadcast()) return false;
+			if (game.IDEA.data) {
+				// @ts-ignore guaranteed above
+				let buf = `<details><summary>IDEA roles:</summary>${game.IDEA.data.roles.join(`<br />`)}</details>`;
```

Comment:
Try to not use `@ts-ignore`, instead try assigning it to a variable and checking that its not null